app/com/arpnetworking/metrics/portal/alerts/impl/DailyPartitionCreator.java

```diff
@@ -187,9 +201,13 @@ public void postStop() throws Exception {
     @Override
     public Receive createReceive() {
         return new ReceiveBuilder()
-                .matchEquals(START_TICKING, msg -> startTicking())
                 .matchEquals(TICK, msg -> tick())
-                .matchEquals(EXECUTE, msg -> execute())
+                .match(CreateForRange.class, msg -> {
+                    final Status.Status resp = execute(msg.getStart(), msg.getEnd());
+                    if (!getSender().equals(getSelf())) {
```

Comment:
Hrm. Is this what you were looking for?

https://doc.akka.io/docs/akka/current/typed/interaction-patterns.html

See: "Ignoring replies"

Reply:
This is exactly what I was looking for, but that's Akka `2.6.XX` using typed actors so it's not available here 😢